app/components/Secrets/Generic/Generic.jsx

```diff
@@ -462,22 +512,22 @@ class GenericSecretBackend extends React.Component {
                                         floatingLabelText="Filter"
                                         hintText="Filter list items"
                                         onChange={(e, v) => {
-                                            this.setState({
-                                                filteredSecretList: _.filter(this.state.secretList, (item) => {
-                                                    return item.includes(v);
-                                                })
-                                            })
+                                            let filtered = v ? _.filter(this.state.fullSecretList, (item) => { return item.includes(v.toLowerCase()); }) : this.state.fullSecretList;
```

Comment:
Interesting, I had thought Vault was case-insensitive, so I figured i'd save on some compute cycles, but apparently thats not the case. Great catch, fix inc.